cms/static/js/views/edit_chapter.js

```diff
@@ -53,7 +53,7 @@ define(["js/views/baseview", "underscore", "underscore.string", "jquery", "gette
             });
             var msg = new FileUploadModel({
                 title: _.template(gettext("Upload a new PDF to “<%= name %>”"),
-                    {name: course.escape('name')}),
+                    {name: course.get('name')}),
```

Comment:
I am a little confused about how escaping will happen if the course's name has a script tag in it. Can you please add a unit test (Jasmine) for this fix, and also verify that XSS is not an issue?